core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -73,23 +76,25 @@ export class EnvOverviewSectionComponent implements OnInit {
 	errorChecked:boolean=true;
 	errorInclude:any="";
   json:any={};
-  slsapp:boolean = false;
   desc_temp:any;
-  toastmessage:any;
+  private toastmessage: any = '';
   copyLink:string="Copy Link";
   disableSave:boolean = true;
- 
+  is_function:boolean;
```

Comment:
is_function is probably no longer needed.

Reply:
Addressed.